Expecto/Expecto.fs

```diff
@@ -618,6 +618,15 @@ module Tests =
     let tests = testFromThisAssembly() |> Option.orDefault (TestList ([], Normal))
     runTestsWithArgsAndCancel ct config args tests
 
+  /// Runs tests in the specified assemblies with the supplied command-line options.
+  /// Returns 0 if all tests passed, otherwise 1
+  let runTestsInAssembliesWithCLIArgsAndCancel (ct:CancellationToken) cliArgs args assemblies =
```

Comment:
This is mostly a style-related comment. :stuck_out_tongue_winking_eye: 

The other functions have the signature written as `let foo (bar : baz) = ...`. Should this one follow the same style?